codegen/src/main/scala/overflowdb/codegen/CodeGen.scala

```diff
@@ -702,7 +772,7 @@ class CodeGen(schema: Schema) {
                    |      case _ => throw new MatchError("unreachable")
                    |    }}.toArray
                    |
-                   |  this._$memberName = if(arr == null) collection.immutable.ArraySeq.empty
+                   |  this._$memberName = if(arr == null) IndexedSeq.empty
```

Comment:
Why this change? I'm not objecting, just asking. 

The original point of me using `ArraySeq.empty` instead of `IndexedSeq.empty` was to maintain an inofficial stronger API (we always return `ArraySeq`), even if we only promise `IndexedSeq` in the type signature.

Reply:
good idea, adjusted in db3d682